cmd/crates/soroban-spec-typescript/fixtures/test_custom_types/src/index.ts

```diff
@@ -2,8 +2,8 @@ import * as SorobanClient from 'soroban-client';
 import { xdr } from 'soroban-client';
 import { Buffer } from "buffer";
 import { scValStrToJs, scValToJs, addressToScVal, u128ToScVal, i128ToScVal, strToScVal } from './convert.js';
-import { invoke, InvokeArgs } from './invoke.js';
-
+import { invoke } from './invoke.js';
```

Comment:
nit: you can drop the `.js` here to match the subsequent line

Reply:
Hmm I remember this being necessary for some targets. So now I'm wondering why the next line worked.

Reply:
The next line probably doesn't work with NextJS; thanks for noticing!